examples/arabic.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+"""
+Example using Arabic
+===============
+Generating a wordcloud from Arabic text
+Other dependencies: bidi.algorithm, arabic_reshaper
```

Comment:
maybe say how to install those. you need to ``pip install python-bidi``, which is not entirely obvious.

Reply:
These are the official pages for both libraries:

https://pypi.python.org/pypi/python-bidi

https://github.com/mpcabd/python-arabic-reshaper

Reply:
http://python-bidi.readthedocs.io/en/latest/ seems more informative

Reply:
I tried this code with huge Arabic data but i get this error:

```
 '%s not allowed here' % _ch['type']

AssertionError: RLI not allowed here
```
Can you resolve it please ?

Reply:
@Abdulrahman44 
I had this problem too when giving large string of arabic text to wordcloud. Notice that the problen is with bidi and not with wordcloud.
I solved it by saving the text to a .docx file, and then opening the file and giving to wordcloud.

Reply:
@Abdulrahman44 
Try removing RLI from the string
[code.docx](https://github.com/amueller/word_cloud/files/1973642/code.docx)

Reply:
@Lotemn102  
How ? can you explain more , please ?

Reply:
> @Lotemn102
> How ? can you explain more , please ?

By replacing every RLI in your text with an empty string.
Have you seen the code.docx file i've added to my previous comment? (can't copy RLI into here)

This problem actually has nothing to do with this repository, so if you have more questions about it, please contact me at lotemn102@gmail.com or try to ask in stackoverflow

Reply:
ok thanks .